packages/now-client/src/utils/index.ts

```diff
@@ -81,17 +81,24 @@ export async function buildFileTree(
   path: string | string[],
   isDirectory: boolean,
   debug: Debug
-): Promise<string[]> {
+): Promise<{ fileList: string[]; ignoreList: string[] }> {
+  const ignoreList: string[] = [];
   let fileList: string[];
-  let { ig } = await getVercelIgnore(path);
+  let { ig, ignores } = await getVercelIgnore(path);
 
-  debug(`Found ${ig.ignores.length} rules in .vercelignore`);
+  debug(`Found ${ignores.length} rules in .vercelignore`);
```

Comment:
This was previously always printing `1` because `ig.ignores` is a function that takes one paramater, so it coincidentally had a `length` property of `1`. Fixed to render the number of rules by checking the proper array.